BlueprintUICommonControls/Sources/Label.swift

```diff
@@ -113,6 +113,12 @@ public struct Label: ProxyElement {
                 // do nothing, use default behavior
                 break
             }
+
+            label.needsTextNormalization = NSAttributedString.needsNormalizingForView(
+                hasLinks: false,
```

Comment:
Hmm good question; I don't think it is – since `UILabel` renders those even outside of `AttributedString` instances; with its own formatting.

Reply:
cc @kylebshr do you recall the answer to (1) above?

Reply:
I think we need normalizing for detected links. UILabel doesn't detect or add links for data links, which is why the detection is done manually. Only UITextView does iirc

Reply:
Ahhh yeah; I forgot we do the checking manually. I think we can check the `linkDetectionTypes`, since it looks like it defaults to `nil` and we don't set it elsewhere until someone explicitly provides it.

Reply:
https://github.com/square/Blueprint/pull/375/commits/3c59fa0b26a2344fc1c781749e35df9cfb8d2c43

Reply:
(Squished things, but the above commit is in the one commit here)

Reply:
Great catch here @robmaceachern.

Reply:
+1, thank you Rob!